### PR TITLE
add missing dataclasses req to setup.py

### PR DESCRIPTION
**Pull request by willzeng**
Found this missing after a fresh `pip install`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,7 +48,8 @@
         'python-rapidjson',
         'pyzmq>=17',
         'ruamel.yaml',
-        'typing'
+        'typing',
+        'dataclasses
```
Comment (stevenheidel) [inline setup.py]: alphabetize, otherwise looks good

Comment (stevenheidel) [reply setup.py]: also missing close quote?

**Comment by mpharrigan** [comment]
consider using [environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) (an idea lovingly stolen from conda)

**Comment by willzeng** [comment]
Good suggestion @mpharrigan I've added that requirement as well.

**Pull request merged**